packages/aws-lambda/lambdas/bin/deploy-zip.sh

```diff
@@ -5,16 +5,7 @@
 # (c) Copyright Instana Inc. and contributors 2019
 #######################################
 
-set -exEo pipefail
-
-if [[ -z "${1-}" ]]; then
-  echo "Usage $0 <lambda-folder-name>"
-  echo
-  echo "The mandatory argument <lambda-folder-name> is missing."
-  exit 1
-fi
-
-cd `dirname $BASH_SOURCE`/../zip
+set -eEo pipefail
 
```

Comment:
I gave the utility love and fixed some bugs. You don't have to review the diff. Its just a test utility.